### ISSUE TITLE
python setup.py build develop --no-deps 

### ISSUE THREAD

**Issue by PanffeeReal** [tianzhi0549/FCOS#368]
Here I encountered a problem as:

$ python setup.py build develop --no-deps
running build
running build_py
running egg_info
writing fcos.egg-info/PKG-INFO
writing dependency_links to fcos.egg-info/dependency_links.txt
writing requirements to fcos.egg-info/requires.txt
writing top-level names to fcos.egg-info/top_level.txt
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'fcos.egg-info/SOURCES.txt'
**error: can't copy 'fcos/configs': doesn't exist or not a regular file**

because focs/configs is a folder instead of a file ,so how to deal with it ? thanks l a lot !

**Comment by PanffeeReal** [comment]
The same problem appears when I use Testing-only installation:

$ pip install git+https://github.com/tianzhi0549/FCOS.git
.......
  error: can't copy 'fcos/configs': doesn't exist or not a regular file
  ----------------------------------------
  ERROR: Failed building wheel for fcos

Help me please,Thank you very much !

**Comment by vorj** [comment]
@PanffeeReal @fsted  
TL;DR: Uninstall `setuptools-scm` .

I found the workaround.
You two must had installed `cocoapi` , and `matplotlib` which is one of dependency for `cocoapi` .
At 2021/11/16 (16days ago), `matplotlib` v3.5.0 has been released, and [`matplotlib` depends on `setuptools-scm`](https://github.com/matplotlib/matplotlib/blob/a4dbe0dbeedd7ca56f8d54dbc7649f81a81e4f51/setup.py#L312) since this version.
I'm not sure of the details, but it seems that this package breaks the behavior of `setuptools` package.
So, the workarounds are below:

1. Uninstall `setuptools-scm` after installing `cocoapi` , or
2. Fix `setup.py` of `cocoapi` to install `matplotlib <= 3.4` , or
3. Install `matplotlib <= 3.4` at first.

If you just want to install and use FCOS, 1st one is the easiest I think.

I have reproduced this problem and have confirmed to fix by either of 1st and 3rd workarounds (I've not checked 2nd one, but it also should work).  
I hope that this post will help you.
